.eslintrc.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     ],
     '@typescript-eslint/no-unused-vars': ['warn', { args: 'none' }],
     '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/camelcase': 'off',
+    '@typescript-eslint/camelcase': 'warn',
```

Comment:
That rule has been removed. You need to set `naming-convention`:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md

Reply:
Not sure we can have both `error` and `warn` for the same `naming-convention` rule?